### PR TITLE
build: add support for windows adhoc signing (bug 1839227)

### PR DESCRIPTION
**Pull request by zzzeid**
- update adhoc-sign script to support windows installer
- rename unsigned installer binary
- add workflow step to print manifest

### CODE REVIEW

File: bin/adhoc-sign.py [hunk 1]
```diff
@@ -53,6 +56,7 @@ def create_parser():
     parser = argparse.ArgumentParser(description="print ad-hoc signing manifest")
     parser.add_argument("release", nargs=1, help="signing manifest release tag")
     parser.add_argument("--bug", default="0", help="optional bug number to include")
+    parser.add_argument("--os", default="macOS", help="os (windows or macOS)")
```
Comment (cgsheeh) [inline bin/adhoc-sign.py:59]: Nit: you could use `choices` here to ensure a valid value is passed..

File: bin/adhoc-sign.py [hunk 2]
```diff
@@ -14,37 +14,40 @@ def main(args: argparse.Namespace):
...
     }
 
     params = {}
 
-    for os, signing_formats in operating_systems.items():
-        url = urls[os]
-        response = requests.get(url)
-        if response.status_code != 200:
-            raise ValueError(f"Could not fetch {url} ({response.status_code})")
+    os = args.os
```
Comment (cgsheeh) [inline bin/adhoc-sign.py:27]: Why is the loop removed?

File: bin/adhoc-sign.py [hunk 3]
```diff
@@ -14,37 +14,40 @@ def main(args: argparse.Namespace):
...
-            "product": "mozregression",
-            "reason": f"Sign application bundle for mozregression {release}.",
-            "requestor": args.requestor,
-            "sha256": hashlib.sha256(response.content).hexdigest(),
-            "signing-formats": signing_formats,
-            "signingscript-notarization": True,
-        }
+    url = urls[os]
+    response = requests.get(url)
+    if response.status_code != 200:
```
Comment (cgsheeh) [inline bin/adhoc-sign.py:32]: Nit: this is a good use case for `raise_for_status`.

**Review by cgsheeh — APPROVED** [review]
LGTM.

File: bin/adhoc-sign.py (hunk 2 above)
Comment (zzzeid) [reply bin/adhoc-sign.py:27]: In practice, this has to be done separately for each OS in different jobs. When I first implemented this I wasn't sure if one manifest can be used or not.

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**